drivers/hid/hid-sony.c

```diff
@@ -1067,8 +1067,24 @@ static int sixaxis_set_operational_usb(struct hid_device *hdev)
 static int sixaxis_set_operational_bt(struct hid_device *hdev)
 {
 	unsigned char buf[] = { 0xf4,  0x42, 0x03, 0x00, 0x00 };
-	return hid_hw_raw_request(hdev, buf[0], buf, sizeof(buf),
-				  HID_FEATURE_REPORT, HID_REQ_SET_REPORT);
+	unsigned char ack;
+	ack = hid_hw_raw_request(hdev, buf[0], buf, sizeof(buf), HID_FEATURE_REPORT, HID_REQ_SET_REPORT);
+	unsigned char enable_gasia[] = {
```

Comment:
Declaring a variable here will cause a compiler warning. It needs to be declared at the beginning with the rest of the variables.